forms-flow-web/src/components/Form/EditForm/FormEdit.js

```diff
@@ -1156,7 +1156,8 @@ const EditComponent = () => {
         nameLabel={t("New Form Name")}
         descriptionLabel={t("New Form Description")}
         showBuildForm={selectedAction === DUPLICATE}
-        isLoading={formSubmitted || validationLoading}
+        isSaveBtnLoading={formSubmitted}
+        isFormNameValidating={validationLoading}
```

Comment:
we will need similar change in List.js page also right ? @Bonymol-aot

Reply:
Missed it while pushing, updated 👍